templates/prelogin/base.html

```diff
@@ -205,6 +205,32 @@ <h3 class="modal-title">
                 </div>
             </div>
         </div>
+        <div id="contactUsFormErrors"
+             class="modal fade">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">{% trans 'Close' %}</span></button>
+                        <h3 class="modal-title">
+                            {% trans "Ooops!" %}
+                        </h3>
+                    </div>
+                    <div class="modal-body">
+                        <p>
+                            {% comment %}
+                                There is only one field that can result in errors,
+                                so it's safe to assume the problem is the e-mail
+                                address.
```

Comment:
Thanks @millerdev. I found [this](https://github.com/dimagi/commcarehq-prelogin/blob/master/forms.py#L90-93), which makes me think @biyeun has a plan for e-mail validation failure. `showEmailError` doesn't appear anywhere else though, so I guess it's probably just a loose end she'll tie up when she gets back. (And this comment might work as a reminder, in case she forgot about it. :grinning: )